app/src/main/res/layout-land/reset_pin_dialog.xml

```diff
@@ -10,21 +10,29 @@
 
   <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:profile="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <org.oppia.android.app.profile.ProfileInputView
+    <com.google.android.material.textfield.TextInputLayout
       android:id="@+id/input_pin"
+      style="@style/TextInputLayout"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_marginStart="24dp"
       android:layout_marginTop="32dp"
       android:layout_marginEnd="24dp"
       android:layout_marginBottom="20dp"
-      app:inputLength="3"
-      app:isPasswordInput="true"
-      app:label="Administrator's PIN"
-      profile:error="@{viewModel.errorMessage}" />
+      app:errorMessage="@{viewModel.errorMessage}">
+
+      <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/input_pin_edit_text"
+        style="@style/TextInputEditText"
+        android:layout_width="match_parent"
```

Comment:
Hint is missing

Reply:
@anandwana001 for this the hint added dynamically